utilities/is-05-control/README.md

```diff
@@ -29,29 +29,38 @@ optional arguments:
   --version VERSION     Version of IS-05 API of DuT
   -s, --sender          Configure NMOS Sender
   -r, --receiver        Configure NMOS Receiver
-  --request REQUEST     JSON data to be sent in the request to configure
-                        sender
-  --sdp SDP             SDP file to be sent in the request to configure
-                        receiver
+  --request REQUEST     JSON data to be sent in the request to configure sender
+  --sdp SDP             SDP file to be queried from a Sender (write) or be sent to a Receiver (read)
   -u UUID, --uuid UUID  UUID of resource to be configured
 ```
 
-Example call to change receiver
+Inside the script the IS-05 device can be controlled with the following keys
 ```
-python3 is05Control.py --ip <hostname or IP> --port <IS-05 Port> --version <IS-05 Version> --receiver --uuid <Receiver ID> --request tune-receiver-to-reference.json --sdp video-1080i-7.sdp
+Press 'e' to set master_enable True
+Press 'd' to set master_enable False
+Press 'c' to set a valid config on a Sender (JSON) or a Receiver(SDP)
+Press 'u' to set a dummy config on a Sender (JSON) or a Receiver(SDP)
```

Comment:
Not sure why it's important to indicate file type, for these one character commands that have no parameters that the user can specify? I think the important point is that the "valid" config means the config specified via `--request` or `--sdp` on the command line, right?

Reply:
right, let's not over-complicate the helper msg.